### ISSUE TITLE
use the newly announced `entries` feature to mark routes automatically as prerenderable

### ISSUE THREAD

**Issue by ivanhofer** [opral/inlang#674]
https://kit.svelte.dev/docs/page-options#entries

We can automatically inject all entry points for all language slugs (`/de`, `/en`, ...).
There is no need for a user to specify all entries in the `svelte.config.js` file.

We need to add the following code to the `[lang]/+page.server.js` file:
```js
import { initState } from '@inlang/sdk-js/adapter-sveltekit/server'

export const entries = async () => {
	const { languages } = await initState(await import('../../../inlang.config.js'))

	return languages.map(language => ({ lang: language }))
}
```

And the following code to the `inlang/[language].json/+server.js` file:
```js
import { initState } from '@inlang/sdk-js/adapter-sveltekit/server'

export const entries = async () => {
	const { languages } = await initState(await import('../../../../inlang.config.js'))

	return languages.map(language => ({ language }))
}
```

We should only add this code if `SvelteKit` > `1.16.0`

**Comment by ivanhofer** [comment]
There is currently a bug in `SvelteKit`: https://github.com/sveltejs/kit/issues/9877 so we should wait

**Comment by ivanhofer** [comment]
The fix was introduced in `1.16.3` so it is the minimum required version we can add the `entries` export.
We need to detect the version of `SvelteKit` when we initialize the plugin.
